src/implementations/twig/components/page-header-core/page-header-core.story.js

```diff
@@ -13,98 +11,106 @@ import dataBreadcrumbLong from '@ecl/specs-component-breadcrumb/demo/data';
 import pageHeaderCore from './page-header-core.html.twig';
 import notes from './README.md';
 
-const preparePageHeaderCore = (data, desc, meta, img) => {
-  data.breadcrumb = dataBreadcrumbLong;
-  data.title = text('title', data.title, tabLabels.required);
-  if (meta) {
-    data.meta = text('meta', data.meta, tabLabels.optional);
+const getArgTypes = (data) => {
+  const argTypes = {};
+  argTypes.title = {
+    name: 'title',
```

Comment:
This is not needed, if your argType key is "title" the control will show "title" even without specifying a "name" for it

Reply:
Updated all argTypes for the 3 page headers ;)